p2p/src/authenticated/actors/listener.rs

```diff
@@ -97,7 +97,7 @@ impl<E: Spawner + Clock + ReasonablyRealtime + Network + Rng + CryptoRng + Metri
         // Reserve also checks if the peer is authorized.
         let peer = incoming.peer();
         let Some(reservation) = tracker
-            .listen(peer.clone())
+            .listen(peer.clone(), incoming.timestamp())
```

Comment:
We will need to track the timestamp on the dialer handshake response as well (someone could respond to our dial with an old handshake message as it doesn't require signing anything based on the handshake message we send).